docs/guides/custom-functions.md

```diff
@@ -16,14 +16,18 @@ export type IFunction<O = any> = (
 ### targetValue
  
 `targetValue` the value the custom function is provided with and is supposed to lint against.
-It's based on `given` JSONPath expression defined on the rule and optionally `field` if placed on `then`.
-For instance, given the following partial of OpenAPI 3.0 document
+
+It's based on `given` [JSONPath][jsonpath] expression defined on the rule and optionally `field` if placed on `then`.
+
+For instance, if `given` has a JSONPath expression of `$`, an the following partial of OpenAPI 3.0 document:
+
 ```yaml
 openapi: 3.0.0
 info:
   title: foo
 ```
-and the following `given` JSONPath expression `$`, `targetValue` would be a JS object literal containing `openapi` and `info` properties.
+
+The  and `targetValue` would be a JS object literal containing `openapi` and `info` properties.
```

Comment:
@philsturgeon Aren't we missing a word (or a couple of words) here between `The` and `and`?

Reply:
Oh damn yeah

Reply:
Fixed, can you re-review?

Reply:
Looks like there’s still something missing between ´The´ and ´and’ 🧐